### PR TITLE
feat(replacements): standard-version to commit-and-tag-version

### PR DESCRIPTION
**Pull request by ST-DDT**
## Changes

Adds a replacement rule for [`standard-version`](https://github.com/conventional-changelog/standard-version/commit/c48005d1e365f960a2262a0a348fbb47a2b640b7) that replaces it with the maintained fork [`commit-and-tag-version`](https://github.com/absolute-version/commit-and-tag-version).

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/config/presets/internal/replacements.ts
Comment (HonkingGoose) [inline lib/config/presets/internal/replacements.ts:905]:
```suggestion
  'standard-version-to-commit-and-tag': {
```

This is more descriptive to me. It also matches the style of other similar presets.

**Comment by ST-DDT** [comment]
I tested the replacement for one of my projects:

- https://github.com/faker-js/faker/pull/2882

`v9.5.0` of `commit-and-tag-version` is a drop in replacement for `standard-version`, later versions rename the command, but are otherwise fully compatible, as you can see in the PR.

File: lib/config/presets/internal/replacements.ts
Comment (ST-DDT) [inline lib/config/presets/internal/replacements.ts:46]:
```suggestion
      'replacements:standard-version-to-commit-and-tag',
```

**Comment by ST-DDT** [comment]
@TimothyJones / maintainers of `absolute-version/commit-and-tag-version`: FYI

File: lib/config/presets/internal/replacements.ts
Comment (HonkingGoose) [inline lib/config/presets/internal/replacements.ts:907]: The [`conventional-changelog/standard-version` readme](https://github.com/conventional-changelog/standard-version/blob/master/README.md) says:

> **`standard-version` is deprecated**. If you're a GitHub user, I recommend [release-please](https://github.com/googleapis/release-please) as an alternative. If you're unable to use GitHub Actions, or if you need to stick with `standard-version` for some other reason, you can use the [commit-and-tag-version]( https://github.com/absolute-version/commit-and-tag-version) fork of `standard-version`.

Is upgrading Renovate users to `commit-and-tag-version` the best thing to do? The maintainer of `standard-version` recommends `release-please`. And the Renovate bot project uses Semantic Release.

It sounds to me that the correct thing to do, as a user of `standard-version` is to move to a tool like `release-please` or Semantic Release?

I'll let the Renovate maintainers decide what to do with my feedback. 😄

Comment (TimothyJones) [reply lib/config/presets/internal/replacements.ts:907]: commit-and-tag-version maintainer here-

Yes; this is a safe thing to do- release please was recommended by the original maintainer of standard-version, but the two tools have different purposes. Release Please is only for users who have GitHub repositories, and additionally it assumes a lot more about how you do your releases.

Semantic release isn’t an appropriate replacement for standard version either - again, the tools do different things. You can’t drop one in for another; you’d have to change your whole release process.

The fork that I maintain was made specifically for those who don’t do their releases in the way that release-please expects, or who are otherwise unable to use github.

Comment (TimothyJones) [reply lib/config/presets/internal/replacements.ts:907]: Thanks for the tag, @ST-DDT !

**Review by TimothyJones — APPROVED** [review]

File: lib/config/presets/internal/replacements.ts
Comment (Shinigami92) [reply lib/config/presets/internal/replacements.ts:907]: Users that did not move away from `standard-version` to `release-please` and ignoring the deprecation error that always shows up by npm are not moving to `release-please` anyway 🤷 
But they might not have heard of `commit-and-tag-version`

File: lib/config/presets/internal/replacements.ts
Comment (viceice) [inline lib/config/presets/internal/replacements.ts:910]:
```suggestion
        matchCurrentVersion: '^9.0.0',
        matchDatasources: ['npm'],
```

Only upgrade users from v9, if somebody uses an older version, then we shouldn't suggest (force) a replacment

**Review by viceice — APPROVED** [review]

**Pull request merged**
